src/app/vault/groupings.component.ts

```diff
@@ -86,4 +90,46 @@ export class GroupingsComponent extends BaseGroupingsComponent {
             organizationId: collection.organizationId,
         });
     }
+
+    async loadCollections(organizationId?: string) {
+        await super.loadCollections(organizationId);
+
+        const organizationWithouKey = await this.localUserService.getOrganizationsWithoutKey();
+
+        organizationWithouKey.forEach(organization => {
+            const collectionCopy = new CollectionView();
+            collectionCopy.id = organization.id;
+            collectionCopy.organizationId = organization.id;
+            const parts = organization.name != null ? organization.name.replace(/^\/+|\/+$/g, '').split(NestingDelimiter) : [];
```

Comment:
Why do we need to replace `///` ?

Reply:
This is because Bitwarden allows to nest folders and collections.
![image](https://user-images.githubusercontent.com/1884255/135091901-c761df7e-11fd-49a0-9480-f3cb9b5d4872.png)

To make this happen, you have to name your folder by specifying the folder structure. For example in the screen you have 4 folders:
- folder1 name: `folder`
- folder2 name: `folder/subfolder1`
- folder3 name: `folder/subfolder2`
- folder4 name: `folder/subfolder2/sub-subfolder`

`ServiceUtils.nestedTraverse()` takes an array corresponding to the decomposed folder path. So we want to split by `NestingDelimiter` (here `/`).
The regex removes leading and trailing `/` if present to prevent adding a folder with empty name.